Tests/BankAccount/Individual/HyperwalletBankAccountIndividualTests.swift

```diff
@@ -14,10 +14,10 @@ class HyperwalletBankAccountTests: XCTestCase {
         }
     }
 
-    func testCreateBankAccount_success() {
+    func testCreateBankAccount_individual_success() {
```

Comment:
Add profile fields such as firstName, lastName etc etc in the test

Reply:
Added missing fields, thanks